### PR TITLE
Better cta readme

### PR DESCRIPTION
**Pull request by epugh**
Working on solving #26 this is very much WIP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+Chorus makes deploying powerful ecommerce search easier by shifting the buy vs build decision in favour of build, so you can own your search! It deals with three issues:
+
+1. **Our collaboration stinks** Making holistic progress on search requires deep, cross-functional collaboration. Shooting emails or tracking search requirements in spreadsheets won't cut it.
+
+2. ***Search testing is hard*** Search changes are cross-cutting: most changes will cause problems. Testing is difficult: you can't run hundreds of searches after every relevance change.
+
+3. **Iterations are slow** Moving forward seems impossible. To avoid sliding backwards, progress is slow. Many simply give up on search, depriving users of the means to find critical information.
+
+
+[Explore a demo](https://fivethirtyeight.datasettes.com/fivethirtyeight), watch [a video about the project](https://www.youtube.com/watch?v=aoWx7KJzvCs) or try it out by [running the quickstart.sh script](#5-minutes-to-run-chorus)
```
Comment (vincetrumental) [inline README.md:21]: This line and next line are from the inspiring README (from datasette project). must be changed for Chorus stuff

Comment (epugh) [reply README.md:21]: Thanks for this comment, I'm going to try and review it now!

Comment (epugh) [reply README.md]: @vincetrumental thoughts on my edits?

File: README.md [hunk 2]
```diff
@@ -1,25 +1,52 @@
...
+
+Want to stay up-to-date with the project? Join the [E-Commerce Search Slack](https://ecom-search.slack.com/) group for tips, tricks and news on what's new in the Chorus ecosystem.
+
+## News
+
+ * 17th June 2021: [Encores? - Going beyond matching and ranking of search results](https://2021.berlinbuzzwords.de/session/encores-going-beyond-matching-and-ranking-search-results) - Chorus is used to demo at BerlinBuzzwords.
+ * 15th November 2020: [Chorus Workshop Series Announced](https://plainschwarz.com/ps-salon/) - Learn from the creators of the components of Chorus via six workshops.
+ * 17th October 2020: [Chorus featured at ApacheCon @Home](https://www.youtube.com/watch?v=NGtmSbOoFjA) - René and Eric give a talk at ApacheCon on Chorus.
+ * 10th June 2020: [Chorus Announced at BerlinBuzzwords](https://2020.berlinbuzzwords.de/session/towards-open-source-tool-stack-e-commerce-search) - First release of Chorus shared with the world at a workshop.
```
Comment (renekrie) [inline README.md:30]: Can we mention the full Gang of Four (you, me, Paul, Johannes) here?

**Review by renekrie — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (epugh) [reply README.md:30]: Yep, adding the origin story.

**Pull request merged**